src/main/java/org/trips/service_framework/heplers/CmsHelper.java

```diff
@@ -38,14 +39,21 @@ public Map<String, Object> createRequestMap(String name, String value, String op
         );
     }
 
-    public Map<String, Object> getSearchQueryFromSkuCode(String code) {
-        Map<String, Object> requestMap = createRequestMap("code", code, "EQ", false);
+    public Map<String, Object> getSearchQueryFromSkuCodes(Collection<String> codes) {
```

Comment:
is the response format same for both the queries?

Reply:
yes

Reply:
why use Collection instead of List? @cf-aagam